xc7/fasm2bels/clb_models.py

```diff
@@ -43,6 +43,29 @@ def create_lut(site, lut):
     return bel
 
 
+def get_srl32_init(features, tile_name, slice_name, srl):
+
+    lut_init = get_lut_init(features, tile_name, slice_name, srl)
+    bits = lut_init.replace("64'b", "")
+
+    return "32'b{}".format(bits[::2])
```

Comment:
If i remember correct bits[1::2] should be equal to bits[::2]?  Maybe add an assertion?

Reply:
I added the assert.